safe/gis/vector/prepare_vector_layer.py

```diff
@@ -464,22 +465,26 @@ def sum_fields(layer, output_field_name, input_fields):
     # Creating expression
     string_expression = ' + '.join(input_fields)
     sum_expression = QgsExpression(string_expression)
-    sum_expression.prepare(layer.pendingFields())
+    context = QgsExpressionContext()
```

Comment:
Is it the correct way to use QgsExpression? Or is it the new way? Just asking since I copied from peta_bencana_dialog.py